### ISSUE TITLE
Wizard: Steps not disabled for keyboard/screenreaders (1.3.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#872]
### Expected behavior

The disabled state information is also available to users who can't see the dimmed UI controls and mouse feedback. Users can't place focus on elements not meant to be in focus order/used and which don't function as expected.

Either links disabled or buttons used similar to modal buttons which are disabled. Button element might be better choice b/c they have disabled attribute. I'm not sure <a> elements can have this attribute/state. 
### Actual behavior

Link disabled state not available programmatically and links are in focus order.
